core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycleSpec.scala

```diff
@@ -275,6 +276,8 @@ class BpmBillingProjectLifecycleSpec extends AnyFlatSpec {
       new CreateLandingZoneResult()
         .landingZoneId(landingZoneId)
     )
+    when(workspaceManagerDAO.deleteLandingZone(landingZoneId, testContext))
+      .thenReturn(new DeleteAzureLandingZoneResult().jobReport(new JobReport().id("fake-id")))
```

Comment:
This test point runs through the cleanup code, so needs to return a `DeleteAzureLandingZoneResult` to avoid NullPointerExceptions.